plugins/Login/lang/en.json

```
@@ -93,6 +93,10 @@
         "LoginFromDifferentCountryEmail6": "We also recommend reviewing your recent account activity to ensure there are no other unauthorised actions.",
         "LoginFromDifferentCountryEmailLinkResetPassword": "Reset your password",
         "LoginFromDifferentCountryEmailLinkEnable2FA": "Enable 2FA",
-        "TimeOfLogin": "Time of login"
+        "TimeOfLogin": "Time of login",
+        "PasswordResetCancelConfirmTitle": "Password reset cancellation",
+        "PasswordResetCancelConfirm": "Confirm password reset cancellation",
+        "PasswordResetCancelConfirmDescription": "You've requested to cancel your password reset. Please confirm your choice to ensure your request is processed. If this was not your intention, you can safely close this page.",
+        "PasswordResetCancelTokenIssue": "We noticed you tried to cancel your password reset request. Unfortunately, this link is no longer valid because it has either expired, invalid or has been used already."
```

The word `invalid` in that sentence feel grammatically incorrect. Should that be:
```suggestion
        "PasswordResetCancelTokenIssue": "We noticed you tried to cancel your password reset request. Unfortunately, this link is no longer valid because it has either expired, is invalid or has been used already."
```